NGitLab.Mock.Tests/NGitLab.Mock.Tests.csproj

```diff
@@ -10,5 +10,6 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.4.1" />
     <PackageReference Include="NUnit" Version="3.13.3" />
     <PackageReference Include="NUnit3TestAdapter" Version="4.3.1" />
+    <PackageReference Include="FluentAssertions" Version="6.7.0" />
```

Comment:
To the other owners (@louis-z , @meziantou), are we ok to add FluentAssertions?

Reply:
I don't have any objections.